src/components/tfoot/tfoot.jsx

```diff
@@ -9,22 +9,39 @@ function Tfoot({
   className,
   children,
   size,
+  variant,
+  hiddenOnMobile,
   theme, // eslint-disable-line react/prop-types
   sheet, // eslint-disable-line react/prop-types
   ...rest
 }) {
-  const usedClassName = classNames(classes.tfoot, size, className);
-  return <tfoot {...rest} className={usedClassName}>{children}</tfoot>;
+  const usedClassName = classNames(
+    classes.tfoot,
+    size,
+    variant ? [variant] : [],
+    { [classes.hiddenOnMobile]: hiddenOnMobile },
+    className
+  );
+
+  return (
+    <tfoot {...rest} className={usedClassName}>
+      {children}
+  </tfoot>
```

Comment:
Something strange with the indentation, check prettier 🤔

Reply:
fixed